src/core/Icon/Icon.tsx

```diff
@@ -27,21 +26,6 @@ export interface IconBaseProps {
 
 export interface IconProps extends IconBaseProps, SuomifiIconInterface {}
 
-export const iconLogger = (
```

Comment:
I agree that we should remove this icon logger. I think that it does not really give as any real value.